Decisions.Microsoft365.Exchange/GraphRest.cs

```diff
@@ -11,7 +11,7 @@ public class GraphRest
     {
         public static HttpResponseMessage HttpResponsePost(ExchangeSettings? settingsOverride, string urlExtension, HttpContent content)
         {
-            return SendHttpRequest(settingsOverride, urlExtension, content, HttpMethod.Post);
+            return CreateHttpRequest(settingsOverride, urlExtension, content, HttpMethod.Post);
```

Comment:
What is the purpose of these changes  vs. just modifying the parameters?

Reply:
CreateHttpRequest will take the URL extension input and combine it with the settings/override base URL, which is how all of the steps are passing it along. See below for further explanation.

Reply:
Right, but can't you do that directly when you send the parameter to SendHttpRequest?

`return SendHttpRequest(settingsOverride, $"{GetSettings(settingsOverride).GraphUrl}{urlExtension}", content, HttpMethod.Post)`

Reply:
Yes, but it seemed redundant to retrieve the settings within each step

Reply:
You're still retrieving it every time you're just doing it in a different method. I guess it's fine.